NAMESPACE

```diff
@@ -139,8 +141,8 @@ export(articulation.points)
 export(articulation_points)
 export(as.directed)
 export(as.igraph)
+export(as.phylo)
 export(as.undirected)
-export(asPhylo)
```

Comment:
this one I really killed.

Reply:
Which seems fine given the occurrences of the same name in https://github.com/search?q=org%3Acran+asPhylo+&type=code are not from this igraph method.